examples/dataset.rml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<TRestDataSet name="BabyIAXO">
+    <parameter name="startTime" value="2022/04/28 00:00"/>
+    <parameter name="endTime" value="2022/11/25 13:20"/>
+    <parameter name="filePattern" value="../../framework-data/pipeline/dataset/BabyIAXO_TrueWolterMicromegasTest*.root"/>
+    <filter metadata="TRestRun::fRunNumber" greaterThan="370" lowerThan="375"/>
+    <filter metadata="TRestRun::fRunTag" contains="Test"/>
+    <observables list="window_transmission:optics_efficiency:axionPhoton_probability:axionPhoton_transmission:boreExitGate_transmission"/>
```

Comment:
why use `:` as the separator? If it's correlated with a vector we can use rml auto parsing, like: `{window_transmission,optics_efficiency,axionPhoton_probability,axionPhoton_transmission,boreExitGate_transmission}`

Reply:
Right, `,` separator is probably much better. I thought about it, perhaps we could use a `Split` automatic separator identifier?

Such as we find on string, if we find only `,`, we use it.

We could use a wildcard Split separator to try to find the separator.

Reply:
Just updated it at [b388349](https://github.com/rest-for-physics/framework/pull/345/commits/b3883496d98e06af576ffb5cbc061da62f430b14)